lib/ret/media_search.ex

```diff
@@ -654,8 +676,8 @@ defmodule Ret.MediaSearch do
       id: hub.hub_sid,
       url: hub |> Hub.url_for(),
       type: :hub,
+      participant_count: RetWeb.Presence.list("hub:#{hub.hub_sid}") |> Enum.count(),
```

Comment:
hmm. this feels a little fishy to me to be grabbing presence here. i think we should push this functionality into the `Ret.Hub` module, and only expose the count as a return value to minimize our interactions with presence, given its sensitivity